disnake/embeds.py

```diff
@@ -268,20 +268,36 @@ def copy(self) -> Self:
         embed._files = self._files.copy()
         return embed
 
-    def __len__(self) -> int:
-        total = len(self.title or "") + len(self.description or "")
-        if self._fields:
-            for field in self._fields:
-                total += len(field["name"]) + len(field["value"])
+    @staticmethod
+    def __len(
+        title: Optional[str],
+        description: Optional[str],
+        fields: Optional[List[EmbedFieldPayload]] = None,
+        footer: Optional[EmbedFooterPayload] = None,
+        author: Optional[EmbedAuthorPayload] = None,
+        strip: bool = True,
+    ):
```

Comment:
why is this a staticmethod that takes internal arguments? Genuinely curious, as it could probably be an instance method which uses the instance attributes, unless there's a reason to be like this?

Reply:
https://github.com/DisnakeDev/disnake/pull/567/commits/229a0dc1215ec6eb197b0d434664bd33c2541601